<legend>About G2Worm: Human GWAS to C Elegans</legend>

<script src="http://code.jquery.com/jquery-latest.js"></script>
<script src="static/bootstrap/js/bootstrap.min.js"></script>

<p><a href="http://massgeneral.org/cedd"><img class="img-border" style="margin-left: 0px; margin-right: 12px; float: right;" src="../static/CEDD-final-CMYK.jpg" alt="CEDD logo" width="348" height="145" /></a></p>
This tool is intended to provide an efficient means of mapping human genes to C elegans, then characterizing phenotypes
which arise from disruption of those genes by shRNA. This might be useful, for example, in designing experiments to further characterize
the biology implicated by human GWAS studies. Data for C elegans phenotypes is drawn from Wormbase via the Wormmart tool. <br>
<hr>
For details about the MGH Psychiatry Center for Experimental Drugs and Diagnostics, see <br>
<a href="http://massgeneral.org/cedd">massgeneral.org/cedd</a>
<br><br>
<legend>Notes</legend>
<div class="accordion" id="accordion2">
<div class="accordion-group">
<div class="accordion-heading">
<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#collapseOne">
General notes
</a>
</div>
<div id="collapseOne" class="accordion-body collapse">
<div class="accordion-inner">
Enter the list of human genes in a tab or comma-separated list in the top text field. Any unmatched genes, either because of
typos or lack of known worm ortholog, will be omitted.
</div>
</div>
</div>
<div class="accordion-group">
<div class="accordion-heading">
<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#collapseTwo">
Interpreting results
</a>
</div>
<div id="collapseTwo" class="accordion-body collapse">
<div class="accordion-inner">
The worm phenotypes vary widely in the number of associated genes - some are specific for 1 or 2 genes, while others are much more
commonly observed (eg, embryonic lethal phenotypes). To aid in interpreting pathway results, the number of (human) genes associated with
a given phenotype are listed in the results. Alternatively, if the 'permute' box is checked, the tool will use permutation to estimate a
p-value - i.e., what is the likelihood that a result as extreme, or more extreme, in terms of number of genes mapping to that pathway
would be observed under the null. The basic unit of analysis here is the gene - the tool as written does not account for LD blocks, for example.
A strength of this approach is that different sources of data can be integrated - for example, genes from GWAS can be combined
with genes/proteins known to be relevant.</div>
</div> 
</div>
<div class="accordion-group>
<div class="accordion-heading">
<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#collapseThree">
Additional notes about permutation
</a>
</div>
<div id="collapseThree" class="accordion-body collapse">
<div class="accordion-inner">
Permutation randomly selects sets of the same number of human genes enterewd in the text box, and determines their pathway mapping.
This process is repeated 1000 times, so for any given pathway, there may be 0 genes appearing x1 times, 1 gene appearing x2 times, 2 genes
appearing x3 times, and so forth. This is a rough indicator of how likely a given result is to arise under the null. Limitations: it does
not fully control for multiple comparisons - each pathway represents a test, though not a fully independent one. Also, it treats every gene
as being equally likely to be selected, when in fact (because of size, location, LD structure, and so forth) this is not the case for GWAS
or sequencing results. Still, this should be a reasonable estimate in most cases.
Important note: If the goal is specifically to test a list of SNPs or CNVs for enrichment, an alternative strategy which *is* aware of genomic structure
(though less able to integrate non-genetic data) is to use either INRICH (Phil Lee) or PLINK (Shaun Purcell). Files generated by g2worm
for use with those tools are available from the author.
</div>
</div>
</div>
</div>
